gateleen-core/pom.xml

```diff
@@ -69,6 +69,10 @@
             <version>${project.version}</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.testng</groupId>
```

Comment:
So, now we have junit and testng on board. Do we really want this?

Reply:
Hmm, not so ideal I agree but

![image](https://user-images.githubusercontent.com/30789476/142155536-aaf5ec14-191c-4fb6-8e23-5ce1c4333057.png)

Reply:
Yeah, testng has a few advantages over junit.
Fine for me, as long as we are aware of this not-so-ideal situation.